.eslintrc.js

```diff
@@ -1,28 +1,20 @@
 module.exports = {
-  extends: 'airbnb-base',
+  extends: ['airbnb-base', 'prettier'],
   env: {
-    'node': true
+    node: true
   },
   rules: {
     // Using 'console' is perfectly okay for a Node.js CLI tool and avoiding
     // it only brings unnecessary complexity
     'no-console': 'off',
 
-    // Node 6 does not support dangling commas in function arguments
-    'comma-dangle': [
-      'error',
-      {
-        'arrays': 'always-multiline',
-        'objects': 'always-multiline',
```

Comment:
I noticed this rule has gone away in some of the diff below. Is the comma dangle behaviour pushed by prettifier or did you make a decision to cut it out?

Reply:
I have explicitly removed it. In combination with `prettier` ESLint config it won't affect anything, delegating proper comma dangling to Prettier.